x-pack/packages/security/authorization_core/src/privileges/privileges.ts

```diff
@@ -247,6 +251,31 @@ export function privilegesFactory(
         }, {}),
       };
     },
+    getFeatureForPrivilege(
```

Comment:
Question: where is this being used? My IDE keeps saying its unused.

Reply:
Good point! Removed in https://github.com/elastic/kibana/pull/197001/commits/42bf34d5d83463d8e40cf5b081fb27534628529d